src/python/WMCore/MicroService/Unified/MSRuleCleaner.py

```diff
@@ -422,21 +426,18 @@ def cleanRucioRules(self, wflow):
         delResults = []
         if self.msConfig['enableRealMode']:
             for rule in wflow['RulesToClean'][currPline]:
-                msg = "%s: Deleting rule['id']: %s "
-                msg += "For did: %s"
-                self.logger.info(msg, currPline, rule['id'], rule['name'])
-                delResult = self.rucio.deleteRule(rule['id'])
+                msg = "%s: Deleting ruleId: %s "
+                self.logger.info(msg, currPline, rule)
+                delResult = self.rucio.deleteRule(rule)
                 delResults.append(delResult)
                 if not delResult:
-                    msg = "%s: Failed to delete rule['id']: %s "
-                    msg += "For did: %s"
-                    self.logger.warning(msg, currPline, rule['id'], rule['name'])
+                    msg = "%s: Failed to delete ruleId: %s "
+                    self.logger.warning(msg, currPline, rule)
         else:
             for rule in wflow['RulesToClean'][currPline]:
                 delResults.append(True)
-                msg = "%s: Is about to delete rule['id']: %s "
-                msg += "For did: %s"
-                self.logger.debug(msg, currPline, rule['id'], rule['name'])
+                msg = "%s: Is about to delete ruleId: %s "
```

Comment:
And same here too.
But here, I would suggest to adopt the same logging format as we have used for the other MS, thus starting the log record with `DRY-RUN: blah blah`

Reply:
Done